src/Fixer/ArrayNotation/NoWhitespaceBeforeCommaInArrayFixer.php

```diff
@@ -74,7 +74,11 @@ private function fixSpacing($index, Tokens $tokens)
             $i = $this->skipNonArrayElements($i, $tokens);
             $currentToken = $tokens[$i];
             $prevIndex = $tokens->getPrevNonWhitespace($i - 1);
-            if ($currentToken->equals(',') && !$tokens[$prevIndex]->equals([T_END_HEREDOC]) && !$tokens[$prevIndex]->isComment()) {
+
+            if (
+                $currentToken->equals(',') && !$tokens[$prevIndex]->isComment() &&
+                (\PHP_VERSION_ID >= 70300 || !$tokens[$prevIndex]->equals([T_END_HEREDOC]))
```

Comment:
hmm, is this the way to go? or do we need a config option for this changed behavior?
Because the travis 7.3 build asks now for cs fixes on this project: https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/474085105#L1299

Reply:
You can add the fixer to https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/.travis.yml#L99

Reply:
@gharlan , there shall be a new config option

Reply:
Ok, I will introduce a config option for this.
And in follow-up prs I will do the same for `TrailingCommaInMultilineArrayFixer` and `MethodArgumentSpaceFixer`.